cmd/influx_inspect/verify/tsm/verify.go

```diff
@@ -34,87 +36,189 @@ func (cmd *Command) Run(args ...string) error {
 	fs := flag.NewFlagSet("verify", flag.ExitOnError)
 	fs.StringVar(&path, "dir", os.Getenv("HOME")+"/.influxdb", "Root storage path. [$HOME/.influxdb]")
 
+	var checkUTF8 bool
+	fs.BoolVar(&checkUTF8, "check-utf8", false, "Verify series keys are valid UTF-8")
+
 	fs.SetOutput(cmd.Stdout)
 	fs.Usage = cmd.printUsage
 
 	if err := fs.Parse(args); err != nil {
 		return err
 	}
 
-	start := time.Now()
 	dataPath := filepath.Join(path, "data")
+	tw := tabwriter.NewWriter(cmd.Stdout, 16, 8, 0, '\t', 0)
+
+	var runner verifier
+	if checkUTF8 {
+		runner = &verifyUTF8{}
+	} else {
+		runner = &verifyChecksums{}
+	}
+	err := runner.Run(tw, dataPath)
+	tw.Flush()
+	return err
+}
+
+// printUsage prints the usage message to STDERR.
+func (cmd *Command) printUsage() {
+	usage := fmt.Sprintf(`Verifies the integrity of TSM files.
+
+Usage: influx_inspect verify [flags]
+
+    -dir <path>
+            Root storage path
+            Defaults to "%[1]s/.influxdb".
+    -check-utf8 
+            Verify series keys are valid UTF-8.
+            This check skips verification of block checksums.
+ `, os.Getenv("HOME"))
 
-	brokenBlocks := 0
-	totalBlocks := 0
+	fmt.Fprintf(cmd.Stdout, usage)
+}
```

Comment:
Can you simply call `fmt.Printf()` instead of `fmt.Sprintf()` -> `fmt.Fprintf()`?